src/api/entities/SecurityToken/Compliance/Requirements.ts

```diff
@@ -69,25 +81,37 @@ export class Requirements extends Namespace<SecurityToken> {
       context
     );
   }
+
+  /**
+   * Add a new compliance requirement to the the Security Token. This doesn't modify existing requirements
+   */
+  public add: ProcedureMethod<AddAssetRequirementParams, SecurityToken>;
+
+  /**
+   * Remove an existing compliance requirement from the Security Token
+   */
+  public remove: ProcedureMethod<RemoveAssetRequirementParams, SecurityToken>;
```

Comment:
Should this one support `Requirement` as param too?

Reply:
Sure, will do